src/main/java/org/gridsuite/merge/orchestrator/server/MergeEventService.java

```diff
@@ -51,11 +51,11 @@ public MergeEventService(MergeRepository mergeRepository, IgmRepository igmRepos
         this.igmRepository = igmRepository;
     }
 
-    public void addMergeIgmEvent(String process, String businessProcess, ZonedDateTime date, String tso, IgmStatus status, UUID networkUuid) {
+    public void addMergeIgmEvent(String process, String businessProcess, ZonedDateTime date, String tso, IgmStatus status, UUID networkUuid, UUID caseUuid) {
```

Comment:
why this change ?

Reply:
Ah ok we store the caseUuid to be able to redownload the original case when we export.

Reply:
Yes